layouts/partials/fragments/item.html

```diff
@@ -97,7 +97,7 @@ <h5>
                     <i class="{{ .Params.icon }} fa-stack-1x fa-inverse"></i>
                   </span>
                 {{- else if .Params.asset -}}
-                  <img src="{{ partial "helpers/image.html" (dict "root" $self "asset" .Params.asset) }}" class="img-fluid p-2" alt="{{ .Params.title }}">
+                  <img src="{{ partial "helpers/image.html" (dict "root" $self "asset" .Params.asset) }}" class="img-fluid p-2" alt="{{ .Params.asset.title }}">
```

Comment:
.Params.asset.text or?

Reply:
Not really sure I didn't name it `.text` in the first place.